docs/section22/Rule22-28.md

```diff
@@ -0,0 +1,39 @@
+
+# CHW&CW - Rule 22-28  
+
+**Schema Version:** 0.0.16  
+**Mandatory Rule:** True  
+**Rule ID:** 22-28  
+**Rule Description:** Each baseline chiller shall be modeled with separate condenser-water pump interlocked to operate with the associated chiller.  
+**Rule Assertion:** B-RMR = expected value  
+**Appendix G Section:** Section 22 CHW&CW Loop  
+**90.1 Section Reference:** Section G3.1.3.11 Heat Rejection (Systems 7, 8, 11, 12, and 13)  
+**Data Lookup:** None  
+**Evaluation Context:** Building  
+**Applicability Checks:**  
+
+1. Baseline ruleset model instance is modeled with at least one air-side system that is Type-7, 8, 11.1, 11.2, 12, 13, 7b, 8b, 11b, 12b, 13b.
+
+**Function Calls:**  
+
+1. get_baseline_system_types()
+
+**Applicability Checks:**  
+
+- Get baseline ruleset model instance system types: `baseline_hvac_system_dict = get_baseline_system_types(ASHRAE229.RulesetModelInstance['BASELINE_0'])`
+
+  - Check if baseline ruleset model instance is modeled with at least one air-side system that is Type-7, 8, 11.1, 11.2, 12, 13, 7b, 8b, 11b, 12b, 13b, i.e. with air-side system served by chiller(s), continue to rule logic: `if any(sys_type in baseline_hvac_system_dict.keys() for sys_type in ["SYS-7", "SYS-8", "SYS-11.1", "SYS-11.2", "SYS-12", "SYS-13", "SYS-7B", "SYS-8B", "SYS-11B", "SYS-12B", "SYS-13B"]): CHECK_RULE_LOGIC`
+
+  - Else, rule is not applicable to baseline ruleset model instance: `else: RULE_NOT_APPLICABLE`
+
+## Rule Logic:
+
+- For each chiller in baseline ruleset model instance: `for chiller_b in ASHRAE229.RulesetModelInstance['BASELINE_0'].chillers:`
+
+  **Rule Assertion:**
+
+  - Case 1. If chiller is interlocked with condenser water pump: `if chiller_b.is_condenser_water_pump_interlocked: PASS`
+
```

Comment:
small correction required here - should check if if chiller_b.is_condenser_water_pump_interlocked = TRUE, then PASS